CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+Thanks for considering contributing and making our planet easier to explore!
+
+We're excited you would like to contribute to Cumulus! Whether you're finding bugs, adding new features, fixing anything broken, or improving documentation, get started by submitting an issue or pull request!
+
+## Submitting an Issue
+
+If you have any questions or ideas, or notice any problems or bugs, first [search open issues](https://github.com/cumulus-nasa/cumulus-message-adapter/issues) to see if the issue has already been submitted. We may already be working on the issue. If you think your issue is new, you're welcome to [create a new issue](https://github.com/cumulus-nasa/cumulus-message-adapter/issues/new).
+
+## Pull Requests
+
+If you want to submit your own contributions, follow these steps:
+
+* Fork the Cumulus Message Adapter repo
+* Create a new branch from the branch you'd like to contribute to
+* If an issue does't already exist, submit one (see above)
+* [Create a pull request](https://help.github.com/articles/creating-a-pull-request/) from your fork into the target branch of the cumulus-nasa/cumulus repo
+* Be sure to [mention the corresponding issue number](https://help.github.com/articles/closing-issues-using-keywords/) in the PR description, i.e. "Fixes Issue #10"
+* Upon submission of a pull request, the Cumulus development team will review the code
+* The request will then either be merged, declined, or an adjustment to the code will be requested
+
+## Guidelines
+
+We ask that you follow these guidelines with your contributions:
+
+### Tests
+
+All of the automated tests for this project need to pass before your submission will be accepted. See the README for instructions on how to run tests and verify that the tests pass. If you add new functionality, please consider adding tests for that functionality as well.
```

Comment:
I would remove "consider" ;)

Reply:
Not my wording, probably Patrick's, so I am going to leave it.